Italian/Kovri-BTCManager-article-italian.md

```diff
@@ -0,0 +1,74 @@
+
+alcune delle domande più interessanti uscite durante l'episodio sono a seguire; *Ipotizziamo che Kovri sia integrato e che Monero abbia portafogli per mobile, quali sono le più grandi sfide a lungo termine che Monero dovrà affrontare?*
+
+SGP ha risposto che, per Monero, le due più grandi sfide sono: rimanere decentralizzato e continuare ad incrementare il livello di privacy. 
+"Non abbiamo nessun sistema, in questo momento, che è perfettamente privato. Dobbiamo continuare ad avvicinarci a questo obbiettivo ". Ha continuato dicendo che, "Siamo a un punto in cui Monero è ragionevolmente privato, adesso dobbiamo farlo diventare ragionevolmente efficente."
+
+A causa delle qualità offuscative, le transazioni di Monero sono più grandi di quelle di Bitcoin, siccome più dati sono sonservati nella blockchain. Diminuire le dimensioni di queste transazioni è una delle priorità chiave del team. Ci sono due sviluppi importanti da questo punto di vista, come evidenziato dai recenti appunti dal [Monero dev meeting](https://monerobase.com/wiki/DevMeeting_2017-08-20). RingCT 2.0 e le ring signatures + CT, che ha il vantaggio aggiunto di non richiedere un set-up fidato.
+
+> "Monero non sarà mai finito, quindi ci sarà sempre più lavoro. Per renderlo più decentralizzato e migliorare la privacy, Immagino Kovri sia un importante aspetto per entrambe le cose." - SGP
+
+Oltre a miglioreare la decentralizzazione e privacy di Monero, Kovri aiuterà anche a dare una spinta al valore e all'adozione di XMR. La gente comincerà a fidarsi della cryptomoneta sempre di più una volta che saranno più fiduciosi nel fatto che ogni transazione li mantiene anonimi. Dopo il rilascio in beta di Kovri, sarà implementato di defalut in Monero, così ogni utente potrà beneficiare dell'extra strato di anonimato I2P.
+
+## La comunità ideologica sta guidando lo sviluppo
+
+Altri punti di discussione dell'episodio di OpenHours sono state le visioni di Spagni sul "consensus" sulla blockchain e i progressi fatti dal lato dello sviluppo, "più interessante del prezzo (di XMR) è la crescita costante dal punto di vista dello sviluppo."
+
+> "Abbiamo visto un flusso di nuovi contributori, alcuni rimangono per poco tempo, ma altri rimangono per molto, e lavorano a qualcosa senza essere pagati. Ci sono alcuni contributori che hanno raccolto fondi tramite il nostro sistema di crowdfunding, ma la maggior parte lavora su Monero perchè gli piace e lo trovano interessante, non perchè pensano che diventeranno ricchi." - Riccardo Spagni
+
+Come Brandon Goodell, [il matematico che è stato preso a bordo di Monero](https://btcmanager.com/no-ico-just-altruism-how-phd-mathematician-future-proof-monero/) per analizzare la cryptomoneta, anche Anonimal è stato finanziato dalla community tramite il sistema di crowdfunding interno al progetto. Ha anche suggerito durante lo show, che i due lavoreranno insieme sullo sviluppo di Kovri nel prossimo futuro.

```

Comment:
che è stato preso a bordo -> che si è unito

Reply:
Thanks, for the review, i implemented your two fixes but i'm not sure about this one. The original translations sound better to me because it's 1- it's the literal translation (consistency) 2- keep the informal tone of the article. If you strongly prefer to use "che si è unito", i can change it